src/platform/EFR32/OTAImageProcessorImpl.cpp

```diff
@@ -174,18 +204,29 @@ void OTAImageProcessorImpl::HandleProcessBlock(intptr_t context)
         return;
     }
 
-    err = bootloader_eraseWriteStorage(mSlotId, mWriteOffset, (uint8_t *) (block.data()), block.size());
-
-    if (err)
+    // Copy data into the word-aligned writeBuffer and once it fills write its contents to the bootloader storage
```

Comment:
what ensures word-alignment here? I would have expected an `__attribute__((aligned (4)));` somewhere

Reply:
The actual change required by the bootloader storage API is that the length of the data written is a multiple of 4, the address alignment is not required. Just the same, I added __attribute__((aligned (4))), doesn't hurt.

Reply:
I was going by the comment: it stated it wants word-aligned write buffer. If it needs a 'buffer sizes of multiple of 4' please update the comment to state that instead. It also seems to make sense to want alignment like for some DMA or something.

Reply:
Done